packages/fxa-settings/src/pages/Signin/SigninUnblock/index.tsx

```diff
@@ -133,7 +133,15 @@ const SigninUnblock = ({
         queryParams: location.search,
       };
 
-      await handleNavigation(navigationOptions, true);
+      const { error: navError } = await handleNavigation(
```

Comment:
I renamed these when we were already in a nested `if (error) {` just for clarity. I could rename this to oAuthError if preferred since that's all it ever is right now but eh.

I'm not loving the error return pattern for cases like this over being able to wrap two functions in a single `try`/`catch`, but I do still like the idea of our GQL call handling matching the GQL query/mutation state shape and returning a handled localized error message... It'd be too much to refactor here anyway.